src/renderer/containers/QuickStart/DolphinMigrationStep.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsx jsx */
+import { ipc_copyDolphin } from "@dolphin/ipc";
+import { DolphinLaunchType } from "@dolphin/types";
+import { css, jsx } from "@emotion/react";
+import styled from "@emotion/styled";
+import Box from "@material-ui/core/Box";
+import Container from "@material-ui/core/Container";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import * as fs from "fs-extra";
```

Comment:
The renderer should not deal with node process things like filesystems. Anything that requires modifying files on the filesystem should dispatch a message to the main process. I think we might still be doing some `fs` stuff still in other renderer components but we really should aim not to.